src/libstd/sys/windows/mod.rs

```diff
@@ -61,7 +61,22 @@ pub fn decode_error_kind(errno: i32) -> ErrorKind {
         c::ERROR_FILE_NOT_FOUND => return ErrorKind::NotFound,
         c::ERROR_PATH_NOT_FOUND => return ErrorKind::NotFound,
         c::ERROR_NO_DATA => return ErrorKind::BrokenPipe,
+        c::ERROR_SEM_TIMEOUT => return ErrorKind::TimedOut,
```

Comment:
Could use or pattern here

Reply:
Just changed and pushed it. I initially decided against using the or pattern, because it isn't used for `ErrorKind::NotFound`, either.

Reply:
Maybe because it is too short to worth using the or pattern.

Reply:
I find it easier to read with the or pattern. Do you think it makes sense to also use it for `NotFound` to ensure uniform style?

Reply:
Sorry, forgot to run `tidy` on the new commit. Fix is in progress ...

Reply:
I don't have a favor. Better let to official reviewer.